index.js

```diff
@@ -0,0 +1,36 @@
+const { parse } = require('csv-parse')
+const fs = require('fs')
+
+function isHabitable (planet) {
+  const planetStatus = planet.koi_disposition
+  const insolationFlux = planet.koi_insol
+  const planetRadius = planet.koi_prad
+  return planetStatus === 'CONFIRMED' &&
+    insolationFlux > 0.36 && insolationFlux < 1.11 &&
+    planetRadius > 1.6
+}
+

```

Comment:
So these numbers are magic numbers. Lets replace them with named variables like:
```suggestion

const MINIMUM_FLUX = 0.36
const MAXIMUM_INSOLATION_FLUX = 1.11
const MINIMUM_PLANET_RADIUS = 1.6

function isHabitable (planet) {
  const planetStatus = planet.koi_disposition
  const insolationFlux = planet.koi_insol
  const planetRadius = planet.koi_prad
  return planetStatus === 'CONFIRMED' &&
    insolationFlux > MINIMUM_FLUX && insolationFlux < MAXIMUM_INSOLATION_FLUX &&
    planetRadius > MINIMUM_PLANET_RADIUS
}


```